roofit/roofitcore/src/RooDataHist.cxx

```diff
@@ -1076,6 +1076,36 @@ RooPlot *RooDataHist::plotOn(RooPlot *frame, PlotOpt o) const
   return RooAbsData::plotOn(frame,o) ;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// A faster version of RooDataHist::weight that assumes the passed arguments
+/// are aligned with the histogram variables.
+///
```

Comment:
Whenever you touch a comment, try to write full doxygen style with documentation of the parameters.
Random example:
https://github.com/root-project/root/blob/bd49faeb7952530b10073df939e329d92ceade95/roofit/roofitcore/src/RooBinSamplingPdf.cxx#L122-L126

I know it's more work, but if you don't do it right now, it won't get done in a long time.

Reply:
Yes, I can do that.